src/main/java/picard/nio/PicardBucketUtils.java

```diff
@@ -1,30 +1,25 @@
 package picard.nio;
 
-import com.google.cloud.storage.contrib.nio.CloudStorageFileSystem;
-import com.google.cloud.storage.contrib.nio.CloudStoragePath;
 import htsjdk.io.IOPath;
 import htsjdk.samtools.util.FileExtensions;
 import htsjdk.utils.ValidationUtils;
+import picard.PicardException;
 
-import java.io.File;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Arrays;
 import java.util.UUID;
 
-
 /**
  * Derived from BucketUtils.java in GATK
  */
 public class PicardBucketUtils {
     public static final String GOOGLE_CLOUD_STORAGE_FILESYSTEM_SCHEME = "gs";
     public static final String HTTP_FILESYSTEM_PROVIDER_SCHEME = "http";
     public static final String HTTPS_FILESYSTEM_PROVIDER_SCHEME = "https";
-    public static final String HDFS_SCHEME = "hdfs";
     public static final String FILE_SCHEME = "file";
 
     // This Picard test staging bucket has a TTL of 180 days (DeleteAction with Age = 180)
-    public static final String GCLOUD_PICARD_STAGING_DIRECTORY = "gs://hellbender-test-logs/staging/picard/";
+    public static final String GCLOUD_PICARD_STAGING_DIRECTORY_STR = "gs://hellbender-test-logs/staging/picard/";
```

Comment:
These really need to be reconciled with the ones in `GCloudTestUtils`, where there is a redundant definition of the staging area root `gs://hellbender-test-logs/staging/`. There really should be only one definition, and all direct usages of this constant should be replaced with calls to the methods in `GCloudTestUtils` that interrogate where the user wants the staging area to be.

Reply:
Thanks for pointing this out. I think it should be handled in a separate PR, if that's alright with you.